unlock-app/src/config.js

```diff
@@ -77,6 +82,7 @@ export default function configure(
   let unlockAddress = runtimeConfig.unlockAddress
   let services = {}
   let supportedProviders = []
+  let unlockUrl = runtimeConfig.unlockUrl || CANONICAL_BASE_DEV_URL
```

Comment:
Not sure we should put env variables in there but only in `next.config.js`?

Reply:
Ha I think I now understand and this is totally my fault. `CANONICAL_BASE_DEV_URL` is a constant and not a env variable :/